### ISSUE TITLE
`-- UNMET PEER DEPENDENCY react@15.4.1

### ISSUE THREAD

**Issue by mithun810** [react-boilerplate/react-boilerplate#1757]
My package.json dependencies are

`"dependencies": {
    "babel-polyfill": "6.20.0",
    "chalk": "1.1.3",
    "compression": "1.6.2",
    "cross-env": "3.1.3",
    "express": "4.14.0",
    "fontfaceobserver": "2.0.7",
    "immutable": "3.8.1",
    "intl": "1.2.5",
    "invariant": "2.2.2",
    "ip": "1.1.4",
    "lodash": "4.17.2",
    "minimist": "1.2.0",
    "react": "^15.4.1",
    "react-dom": "^15.4.1",
    "react-helmet": "3.2.2",
    "react-intl": "2.1.5",
    "react-native": "^0.44.0",
    "react-native-material-design": "^0.3.7",
    "react-redux": "4.4.6",
    "react-router": "3.0.0",
    "react-router-redux": "4.0.6",
    "react-router-scroll": "0.4.1",
    "redux": "3.6.0",
    "redux-immutable": "3.0.8",
    "redux-saga": "0.14.0",
    "reselect": "2.5.4",
    "sanitize.css": "4.1.0",
    "styled-components": "1.1.2",
    "warning": "3.0.0",
    "whatwg-fetch": "2.0.1"
  },`

**Comment by mithun810** [comment]
@mxstbr Any help here?

**Comment by IamJoseph** [comment]
Hey man... could you copy / paste the full error?

**Comment by mithun810** [comment]
+-- UNMET PEER DEPENDENCY react@15.4.1                                                                                                                                     
`-- UNMET PEER DEPENDENCY webpack@2.2.0-rc.3                                                                                                                               
                                                                                                                                                                           
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules/chokidar/node_modules/fsevents):                                                             
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.1: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})          
npm WARN eslint-import-resolver-webpack@0.8.0 requires a peer of webpack@^1.11.0 || ^2.1.0-beta || ^2.1.0 but none was installed.                                          
npm WARN html-webpack-plugin@2.24.1 requires a peer of webpack@1 || ^2.1.0-beta but none was installed.

**Comment by IamJoseph** [comment]
You didn't list your devDependencies so I'm unsure what version of webpack you're using but it looks like these are just warnings. If you want them to go away just update them. You can see all your outdated packages by typing npm outdated. AFAIK you can update most of them sans react-router and image-webpack-loader without too much trouble. Image-webpack-loader is an easy fix but react-router would take awhile.

**Comment by carlfairclough** [comment]
I was having the same issue with a fresh install.

I had a very old version of Node.js installed. I updated that and nuked `node_modules`. After that, I was receiving errors because `/internals/scripts/setup.js` is deleted after you run `npm run setup` for the first time.

Cloning the repo again fixed it, but I suspect that just restoring `/internals/scripts/setup.js` would also resolve the issue.

So, in summary:
 - Make sure you have the latest stable version of Node.js
 - Delete `node_modules`
 - Make sure you have `internals/scripts/setup.js`
 - Run `npm run setup`
